.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,62 @@
+# based on https://github.com/nfriedly/gmenunx/blob/ci/.github/workflows/c-cpp.yml
+name: CI Build
+
+on: [push, pull_request]
+
+jobs:   
+  build-modern:
+    name: MiyooCFW Linux kernel (musl libc)
+    runs-on: ubuntu-20.04
+    container:
+      image: nfriedly/miyoo-toolchain:latest
+    steps:
+    - uses: actions/checkout@v2
+    # ARCH=arm and CROSS_COMPILE=arm-buildroot-linux-musleabi- env props are set in the docker image
+    - name: build devicetree
```

Comment:
Oh, shoot, this line is wrong. It should say configure or something like that. I corrected it on my laptop, but then I forced pushed over it from my desktop 🤦